### PR TITLE
Removed solicitor duplicated state

### PR DESCRIPTION
**Pull request by qzhou-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/divorce/json/AuthorisationCaseState.json [hunk 1]
```diff
@@ -1336,41 +1336,13 @@
     "UserRole": "caseworker-divorce-solicitor",
     "CRUD": "R"
   },
-  {
-    "LiveFrom": "25/06/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseStateID": "AwaitingDecreeNisi",
-    "UserRole": "caseworker-divorce-solicitor",
```
Comment (qzhou-hmcts) [inline definitions/divorce/json/AuthorisationCaseState.json]: Already added on https://github.com/hmcts/div-ccd-definitions/blob/12f97c9b53e605089eed2828396ef87df065bfeb/definitions/divorce/json/AuthorisationCaseState.json#L1349

File: definitions/divorce/json/AuthorisationCaseState.json [hunk 2]
```diff
@@ -1336,41 +1336,13 @@
...
-    "CRUD": "RU"
-  },
-  {
-    "LiveFrom": "25/06/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseStateID": "AosSubmittedAwaitingAnswer",
-    "UserRole": "caseworker-divorce-solicitor",
```
Comment (qzhou-hmcts) [inline definitions/divorce/json/AuthorisationCaseState.json]: Already added on https://github.com/hmcts/div-ccd-definitions/blob/12f97c9b53e605089eed2828396ef87df065bfeb/definitions/divorce/json/AuthorisationCaseState.json#L1357

File: definitions/divorce/json/AuthorisationCaseState.json [hunk 3]
```diff
@@ -1336,41 +1336,13 @@
...
     "LiveFrom": "25/06/2019",
     "CaseTypeID": "DIVORCE",
     "CaseStateID": "DNDrafted",
     "UserRole": "caseworker-divorce-solicitor",
     "CRUD": "CRU"
   },
-  {
-    "LiveFrom": "25/06/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseStateID": "AosOverdue",
```
Comment (qzhou-hmcts) [inline definitions/divorce/json/AuthorisationCaseState.json]: Already added on https://github.com/hmcts/div-ccd-definitions/blob/12f97c9b53e605089eed2828396ef87df065bfeb/definitions/divorce/json/AuthorisationCaseState.json#L1328

File: definitions/divorce/json/AuthorisationCaseState.json [hunk 4]
```diff
@@ -1336,41 +1336,13 @@
...
-    "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "R"
-  },
-  {
-    "LiveFrom": "25/06/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseStateID": "AwaitingLegalAdvisorReferral",
```
Comment (qzhou-hmcts) [inline definitions/divorce/json/AuthorisationCaseState.json]: Already added on . https://github.com/hmcts/div-ccd-definitions/blob/12f97c9b53e605089eed2828396ef87df065bfeb/definitions/divorce/json/AuthorisationCaseState.json#L1335

**Comment by qzhou-hmcts** [comment]
> Can we have a simple unit test for this please? :)
> i.e - check the file and fail the test if there is permission appearing twice (we have similar tests for case tab)

Do you mean a test to validate the config? It sounds like a techdeb ticket

**Comment by hosainnet** [comment]
> Do you mean a test to validate the config? It sounds like a techdeb ticket

Not the whole config, just this area you're fixing :)
We can have a tech debit but if it's a beneficial and won't take too long then why not here ;)

**Comment by qzhou-hmcts** [comment]
> > Do you mean a test to validate the config? It sounds like a techdeb ticket
> 
> Not the whole config, just this area you're fixing :)
> We can have a tech debit but if it's a beneficial and won't take too long then why not here ;)

If you think this is something missing  in the project is not better to have a proper ticket and add it through  refinement and planning instead of sneaky in different PR's?

**Comment by hosainnet** [comment]
> > > Do you mean a test to validate the config? It sounds like a techdeb ticket
> > 
> > 
> > Not the whole config, just this area you're fixing :)
> > We can have a tech debit but if it's a beneficial and won't take too long then why not here ;)
> 
> If you think this is something missing in the project is not better to have a proper ticket and add it through refinement and planning instead of sneaky in different PR's?

Sure we can do it that way too - I don't usually do that when I create a junit test  thought :D

**Comment by qzhou-hmcts** [comment]
@yarpo  I've created this ticket to cover the config validation https://tools.hmcts.net/jira/browse/DIV-5490 prioritise as your convenience

**Comment by hosainnet** [comment]
> @yarpo I've created this ticket to cover the config validation https://tools.hmcts.net/jira/browse/DIV-5490 prioritise as your convenience

@yarpo any chance we can bring this into the sprint? Just spent half an hour trying to find which fields/auth are duplicated - CCD simply showed this and had to wait for logs to figure out one by one 
Screenshot 2019-07-24 at 15 03 49

**Pull request merged**